packages/eddsa-ticket-pcd/src/EdDSATicketPCD.ts

```diff
@@ -147,6 +190,10 @@ export async function prove(args: EdDSATicketPCDArgs): Promise<EdDSATicketPCD> {
   return new EdDSATicketPCD(id, { ticket: args.ticket.value }, { eddsaPCD });
 }
 
+/**
+ * Verifies an EdDSA Ticket PCD by checking that its {@link EdDSATicketPCDClaim} corresponds to
+ * its {@link EdDSATicketPCDProof}. If they match, the function returns true, otherwise false.
```

Comment:
worth adding a warning here that further checks need to be performed by users of this function to make sure that the ticket was signed by the right entity and that its event and product details match what is needed/expected.